src/firefox/lib/glue.js

```diff
@@ -170,6 +180,58 @@ function setUpConnection(freedom, panel, button) {
       });
     }
   });
+
+
+  // Check if user already has a tab open to the uProxy install page.
+  for (var tab of tabs) {
+    if (isInstallPage(tab.url)) {
```

Comment:
I mean, worst case for these, we could trim the last character off and check if the string begins with the rest, not any more complicated than the current function we have.